eth/api.go

```diff
@@ -1376,6 +1376,140 @@ func (api *PublicDebugAPI) SeedHash(number uint64) (string, error) {
 	return fmt.Sprintf("0x%x", hash), nil
 }
 
+
+type StructLogRes struct {
+	Pc      uint64            `json:"Pc"`
```

Comment:
If you leave the field capitalized in the json tag, then it's not needed as that's the default interpretation. However, to conform to the existing APIs, please don't capitalize the first letters.